eclass/gnustep-base.eclass

```diff
@@ -1,17 +1,18 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 # @ECLASS: gnustep-base.eclass
 # @MAINTAINER:
 # GNUstep Herd <gnustep@gentoo.org>
-# @SUPPORTED_EAPIS: 5 6 7 8
+# @SUPPORTED_EAPIS: 6 7 8
 # @BLURB: Internal handling of GNUstep pacakges
 # @DESCRIPTION:
 # Inner gnustep eclass, should only be inherited directly by gnustep-base
 # packages
 
-case ${EAPI:-0} in
-	[5678]) inherit eutils ;;
+case ${EAPI} in
+	6|7) inherit eutils ;;
+	8) ;;
 	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
 esac
 
```

Comment:
Huh. GNUSTEP_CORE_DEPEND isn't used anywhere in `eclass/`?

Reply:
It's used in ebuilds. Also, how is that related to this PR?

Reply:
Reviewing the PR requires checking the whole eclass for uses of eutils. I'd be remiss if I didn't then note something odd which might then be a missing dependency.

I guess it should have an eclassdoc tag then but out of scope for this.

It doesn't mean you need to address it. If you hadn't explained it, voyageur would have.

Reply:
In any case, there's no large incentive for pushing changes together, because gnustep-base is inherited only by some 70 ebuilds (i.e. the impact on the metadata cache is limited).

Reply:
Nice catch sam (and thanks for checking the whole ebuild), I guess some parts of these eclasses show their age, and +1 on GNUSTEP_CORE_DEPEND having a tag beyond the current comment (not related to this eutils change)